decidim-meetings/app/assets/javascripts/decidim/meetings/meetings_filters.js.es6

```diff
@@ -0,0 +1,19 @@
+// = require ./meetings_form_filter.component
+// = require_self
+
+((exports) => {
+  const { DecidimMeetings: { MeetingsFormFilterComponent } } = exports;
+  const meetingsFormFilter = new MeetingsFormFilterComponent('form#new_filter');
+
+  const onDocumentReady = () => {
+    meetingsFormFilter.mountComponent();
+  };
+
+  const onTurboLinksBeforeVisit = () => {
+    meetingsFormFilter.unmountComponent();
+    $(document).off('turbolinks:before-visit', onTurboLinksBeforeVisit);
+  };
+
+  $(document).ready(onDocumentReady);
```

Comment:
This code still feels very strange to me... I'm not really sure why we need to mix document events with turbolinks ones. Can you add some documentation about this so we remember why we didn't use `turbolinks:load` event instead of document ready event.